openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/TemplateOperationsImpl.java

```diff
@@ -202,6 +205,12 @@ public KubernetesList processLocally(Map<String, String> valuesMap)  {
             } else {
               throw new IllegalArgumentException("No value available for parameter name: " + parameterName);
             }
+            if (parameterValue == null) {
+              if (logger.isDebugEnabled()) {
```

Comment:
What's the point to checking whether debug is enabled or not when you're already doing logger.debug?